salt/minion.py

```diff
@@ -887,8 +887,9 @@ def _process_beacons(self):
         # Process Beacons
         try:
             beacons = self.process_beacons(self.functions)
-        except Exception:
-            log.critical('The beacon errored: ', exec_info=True)
+        except Exception as exc:
```

Comment:
It was only giving me the last line and not the full traceback. Didn't investigate why but just replaced it instead. *shrug*